src/powershell/enable-rdp.ps1

```diff
@@ -28,3 +28,40 @@ if ($(Get-NetFirewallRule -Name $name).Enabled -ne $true) {
     Write-Error "[X] Failed to verify firewall rule: $name" -ErrorAction Stop
 }
 Write-Output "[*] Firewall rule successfully verified: $name"
+
+# Allow Terminal Server shadowing through the Windows firewall
+$name = "Remote Desktop - Shadow (TCP-In)"
+Write-Output "[ ] Configuring firewall rule: $name"
+Set-NetFirewallRule -DisplayName $name -Enabled True -Profile Private
+if ($(Get-NetFirewallRule -DisplayName $name).Enabled -ne $true) {
+    Write-Error "[X] Failed to verify firewall rule: $name" -ErrorAction Stop
+}
+Write-Output "[*] Firewall rule successfully verified: $name"
+
+# Enable Terminal Server shadowing
+# Set value to option 2:
+# "Configure Group Policy setting to not require the user’s consent"
+$name = "Shadow"
+$path = 'HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp'
+Write-Output "[ ] Configuring setting: $name"
+Set-ItemProperty -Path $path -Name $name -Value "2"
+if ($(Get-ItemProperty -Path $path -Name $name).Shadow -ne 2) {
+    Write-Error "[X] Failed to verify setting: $name" -ErrorAction Stop
+}
+Write-Output "[*] Setting successfully verified: $name"
+
+# Give the Administrators group full control of Terminal Server
```

Comment:
It would be nice if there was some descriptive output around this part. Even better if the change can be verified so you can have consistent `Doing x -> Failed to do x/Verified x` messaging.

Reply:
@mcdonnnj Just a quick update, I'm working through understanding this command. I'm having a feeling this command is not actually setting anything and might not be needed. I'll let you know once I find out.

Reply:
@mcdonnnj after some digging around. I was able to get more context on this step and I updated it to include error handling and a more descriptive comment. Resolved with [391d2cc](https://github.com/cisagov/windows-server-packer/pull/22/commits/391d2cca40e640345cd4a0639d16833ffd253536)